blueprint/blueprint-core/src/main/java/org/apache/aries/blueprint/container/BlueprintExtender.java

```diff
@@ -95,7 +96,14 @@ public void start(BundleContext ctx) {
         handlers = new NamespaceHandlerRegistryImpl(trackingContext);
         executors = new ScheduledExecutorServiceWrapper(ctx, "Blueprint Extender", new ScheduledExecutorServiceFactory() {
             public ScheduledExecutorService create(String name) {
-                return Executors.newScheduledThreadPool(3, new BlueprintThreadFactory(name));
+                int extenderThreads = 10;
```

Comment:
We should keep the default of 3. So people who do not change it do not see a different behaviour.